packages/opentelemetry/src/utils/parseSpanDescription.ts

```diff
@@ -162,6 +171,22 @@ export function descriptionForHttpMethod(
   };
 }
 
+function getGraphqlOperationNames(attr: AttributeValue): string {
+  if (Array.isArray(attr)) {
+    const sorted = attr.slice().sort();
+
+    // Up to 5 items, we just add all of them
```

Comment:
h: This logic shows up to 4 items and when it is five we show "a, b, c, d, e, +0".

Reply:
oops, great catch :D